src/full/Agda/Compiler/Treeless/Unused.hs

```diff
@@ -43,8 +43,8 @@ computeUnused q t used = do
       TLam b    -> underBinder <$> go b
       TLet e b  -> do
         uses <- go b
-        if | Set.member 0 uses -> Set.union (underBinder uses) <$> go e
-           | otherwise         -> pure (underBinder uses)
+        if Set.member 0 uses then Set.union (underBinder uses) <$> go e
```

Comment:
What's wrong with the multi-way if?

Reply:
I must admit that it's a matter of taste so happy to go with the original, but my reasoning is that I prefer compact code (then+else v | *2 + -> *2+otherwise).